core/templates/dev/head/player/flag_exploration_popup.html

```diff
@@ -0,0 +1,42 @@
+<script type="text/ng-template" id="modals/flagExploration">
+  <div class="modal-header">
+    <h3>
+      <span>Report Exploration</span>
```

Comment:
Changed to Report this exploration.
I am not sure if I can use google translate to internationalize this. I tried that on tooltip. Depending on review of that I can internationalize this.